projects/frontend/data-pipelines/gui/version.txt

```diff
@@ -1 +1 @@
-0.0.0
+1.0.0-alpha.0
```

Comment:
You may review the versioning again in the component docs?

Reply:
I could change it to `1.0.0-alpha.1` which is valid according the semver semantic.
Am I doing something wrong according the code standards in VDK?

Reply:
Changed.